libraries/botframework-connector/tests/auth.test.js

```diff
@@ -1,434 +0,0 @@
-const assert = require('assert');
```

Comment:
This was broken up into several discrete tests according to the class under test.